### ISSUE TITLE
Late action execution

### ISSUE THREAD

**Issue by Ch1pStar** [jiborobot/pixi-animate#56]
When a frame with an action is skipped, the action is executed on the next possible frame. This works fine, but there is a problem with movieclip properties about the current frame.

My exact case is the following. There are multiple animations for a movieclip on the timeline, separated by labels. When a specific animation is required to be played, it is supposed to stop at its end with `this.stop()`, but when a frame is skipped the animation stops a few frames later(which is a completely different state for the movieclip).

My current workaround solution is to have an animation end label and another keyframe at the end with the action `this.gotoAndStop('animation_end_label')`, but this is a very hacky solution and adds extra complexity to the .fla resources.

**Comment by bigtimebuddy** [comment]
Someone's else brought this up recently and a related issue to sounds being triggered. 

My current idea is to actually remove frame-dropping as the default and make it opt-in. That way every frame is played and it's the responsibility of the user to manage their performance issues. This is actually how Flash works, with the exception of streaming syncing sounds.

**Comment by Ch1pStar** [comment]
@bigtimebuddy Is something like this what you have in mind?
https://github.com/Ch1pStar/pixi-animate/commit/abdd2b91678fdf6c1f8e3b7f69271ad375c981b5

**Comment by bigtimebuddy** [comment]
Exactly! The only thing I’d change is to make a frameSkip non-private so it can be toggled in and off.
